src/vs/monaco.d.ts

```diff
@@ -2547,6 +2547,14 @@ declare namespace monaco.editor {
 		 * Enable using global storage for remembering suggestions.
 		 */
 		shareSuggestSelections?: boolean;
+		/**
+		 * Enable or disable icons in suggestions. Defaults to true.
+		 */
+		showIcons?: boolean;
+		/**
+		 * Max suggestions to show in suggestions. Defaults to 12.
+		 */
+		maxSuggestionsToShow?: boolean;
```

Comment:
Are both JSDocs are correct?

Reply:
@jrieken Already pushed new commit. Let me know if a new implementation is correct.